assets/js/search-data.js

```diff
@@ -21,6 +21,7 @@
     'id': {{ $index }},
     'href': '{{ $page.RelPermalink }}',
     'title': {{ (partial "title" $page) | jsonify }},
+    'parent': {{ with $page.Parent }}{{ (partial "title" . ) | jsonify }}{{ else }}""{{ end }},
```

Comment:
use single quotes and remove whitespace

`{{ with $page.Parent }}{{ (partial "title" .) | jsonify }}{{ else }}''{{ end }},`